### ISSUE TITLE
Canvas: trying to use a recycled bitmap android.graphics.Bitmap

### ISSUE THREAD

**Issue by rasmikantaguru** [wunderlist/android-sliding-layer-lib#35]
Hi,

I am getting the below error.I am not able to solve it. 
Can anyone help me.

java.lang.RuntimeException: Canvas: trying to use a recycled bitmap android.graphics.Bitmap

**Comment by shijilsmb** [comment]
I have the same issue

**Comment by mhemdan** [comment]
I have the same issue Too

**Comment by JlUgia** [comment]
Alright, so with HW accel it works for all? Let's gather information about the versions where this is happening so that we can focus our tests on those and propose an alternate solution. There are many variances on how the OS enabled/disabled hardware accel based on different circumstances, especially on the transition to ICS and JB.

**Comment by mhemdan** [comment]
Sloved after adding this parameter on the manifest.xml 
    <application
        android:hardwareAccelerated="true"
        android:largeHeap="true">
Thanks

**Comment by JlUgia** [comment]
Yeah, still would be great to avoid devs facing this issue. When it doesn't work on software we should probably force hardware.

**Comment by nikunjparadva** [comment]
solved it @JlUgia  
by adding ****`android:hardwareAccelerated="true"`****

Thanks man

**Comment by JlUgia** [comment]
And this one looks like a dupe.

**Comment by jsquarede** [comment]
not working

**Comment by jsquarede** [comment]
please give solution fast
